test/integration/node-specific/examples/versioned_api.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const { MongoClient } = require('../../../src');
+const { MongoClient } = require('../../../../src');
```

Comment:
Can we rename this file to end in `.test.js`?

Reply:
True it would need to skip based on `MONGODB_API_VERSION` being defined, we don't need to do that here.